CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to Open MPI
+
+First off, thank you for taking the time to prepare a contribution to
+Open MPI!
+
+![You're awesome!](https://www.open-mpi.org/images/youre-awesome.jpg)
+
+## Content
+
+We love getting contributions from anyone.  But keep in mind that Open
+MPI is used in production environments all around the world.
+
+If you're contributing a small bug fix, awesome!
+
+If you're contributing a large new piece of functionality, that will
+be best viewed if you -- or someone, anyone -- is also stepping up to
+help maintain that functionality over time.  We love new ideas and new
+features, but we do need to be realistic in what we can reliably test
+and deliver to our users.
+
+## Contributor's Declaration
+
+In order to ensure that we can keep distributing Open MPI under our
+[open source license](LICENSE), we need to ensure that all
+contributions are compatible with that license.
+
+To that end, we require that all Git commits contributed to Open MPI
+have a "Signed-off-by" token indicating that the commit author agrees
+with [Open MPI's Contributor's
+Declaration](https://github.com/open-mpi/ompi/wiki/Admistrative-rules#contributors-declaration).
+
+If you have not already done so, please ensure that:
+
+1. Every commit contains exactly this token.  You can add this token
```

Comment:
Change `this token` to `the "Signed-off-by" token`

Reply:
Done!